content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,315 @@
 
 .. tip::
-   To save time on a *call for tender*, custom vendors, prices, and delivery lead times can be set
-   in the :guilabel:`Purchase` tab on a product form. To do so, navigate to
-   :menuselection:`Purchase --> Products --> Products`, and select a product to edit. From the
-   product form, click the :guilabel:`Purchase tab`, then click :guilabel:`Add a line`. From the
-   drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, and set a
-   :guilabel:`Price` and :guilabel:`Delivery Lead Time` if desired. Clicking the
-   :guilabel:`additional options (two-dots)` icon provides additional visibility options to add to
+   To save time creating alternative |RfQs|, custom vendors, prices, and delivery lead times can be
+   set in the :guilabel:`Purchase` tab of a product form.
+
+   To do that, navigate to :menuselection:`Purchase app --> Products --> Products`, and select a
+   product to edit. On the product form, click the :guilabel:`Purchase` tab, and click
+   :guilabel:`Add a line`.
+
+   From the drop-down menu, choose a vendor to set under the :guilabel:`Vendor` column, then set a
+   :guilabel:`Price` and :guilabel:`Delivery Lead Time`, if necessary. Clicking the
+   :guilabel:`(optional columns drop-down toggle)` icon provides additional column options to add to
    the line item.
 
-Create an :abbr:`RFQ (Request for Quotation)`
-=============================================
+.. _purchase/manage_deals/create-rfq:
+
+Create an |RfQ|
+===============
+
+To create a new |RfQ|, navigate to the :menuselection:`Purchase` app, and click :guilabel:`New`.
 
-To create a new :abbr:`RFQ (Request for Quotation)`, navigate to the :menuselection:`Purchase` app,
-and click :guilabel:`New`.
+On the blank |RfQ| form, add a vendor from the drop-down menu, next to the :guilabel:`Vendor` field.
+Then, click :guilabel:`Add a product`, in the :guilabel:`Products` tab, to select a product from the
+drop-down menu in the :guilabel:`Product` column.
 
-Then, add information to the :abbr:`RFQ (Request for Quotation)` form: add a vendor from the
-drop-down next to the :guilabel:`Vendor` field, and click :guilabel:`Add a product` to select a
-product from the drop-down menu in the :guilabel:`Product` column. Then, set the desired purchase
-quantity in the :guilabel:`Quantity` column, and change the purchase price in the :guilabel:`Unit
-Price` column, if desired.
+Then, set the desired quantity to purchase in the :guilabel:`Quantity` column, and change the
+purchase price in the :guilabel:`Unit Price` column, if necessary.
 
-Clicking the :guilabel:`additional options (two-dots)` icon provides additional visibility options
-to add to the line item. Repeat these steps to add as many options as desired, including the
-:guilabel:`UoM` (Units of Measure) to purchase the products in, and the :guilabel:`Expected Arrival`
-date.
+Clicking the :guilabel:`(optional columns drop-down toggle)` icon provides additional column options
```

Comment:
same note as above

Reply:
Added clarification for where to find this icon. Suggestion implemented ✅